src/EnergyPlus/Autosizing/Base.hh

```diff
@@ -116,7 +116,7 @@ namespace EnergyPlus {
         Array1D<EnergyPlus::DataSizing::SystemSizingData> finalSysSizing;
         std::vector<AirLoopHVACDOAS::AirLoopDOAS> airloopDOAS;
 
-        virtual void initializeWithinEP(EnergyPlusData &state, std::string const &_compName, std::string const &_compType, bool printWarningFlag) = 0;
+        virtual void initializeWithinEP(EnergyPlusData &state, std::string const &_compType, std::string const &_compName, bool _printWarningFlag);
```

Comment:
Unified the argument order.  A couple were type then name, and others were name then type.